docs/rules/no-extra-parens.md

```diff
@@ -23,6 +23,7 @@ This rule has an object option for exceptions to the `"all"` option:
 * `"conditionalAssign": false` allows extra parentheses around assignments in conditional test expressions
 * `"returnAssign": false` allows extra parentheses around assignments in `return` statements
 * `"nestedBinaryExpressions": false` allows extra parentheses in nested binary expressions
+* `"ignoreJSX": "all|multiline|single-line"` allows extra parentheses around all/multiline/single-line JSX components
```

Comment:
Which of these three choices represents the default? It seems like we'd want a "none" option - that way I wouldn't be forced to delete the key (I like documenting all possible options even if I don't use them). Could we add a "never"?